dev/test/query.ts

```diff
@@ -285,9 +286,24 @@ function endAt(
   return {endAt: cursor};
 }
 
-export function queryEqualsWithParent(
+/**
+ * Returns the timestamp value for the provided readTimes, or the default
+ * readTime value used in tests if no values are provided.
+ */
+export function readTime(
+  seconds?: number,
+  nanos?: number
+): protobuf.ITimestamp {
+  if (seconds === undefined && nanos === undefined) {
+    return {seconds: '5', nanos: 6};
```

Comment:
We have to cast to seconds since the `Timestamp` class always [converts it](https://github.com/googleapis/nodejs-firestore/blob/5f6683c2177aa7bd80ec81c3942443453b834529/dev/src/timestamp.ts#L262) to `string` for some reason. Why do we do this?

Reply:
Proto3 JSON represents int64 as strings, and the seconds component of Timestamps is an int64.